dri/report.py

```diff
@@ -56,11 +63,13 @@
     'azure/azure-cli',
 ]
 
+# Do not apply user filters to these repos.
 BYPASS_USERFILTER_REPOS = [
     'botbuilder-tools',
 ]
 
-MICROSFT_EMPLOYEES=[
```

Comment:
Perhaps we rename this to MSFT_EMPLOYEES so it doesn't look like a misspell? :-)